### ISSUE TITLE
Convertir el parseador del PDF a JavaScript

### ISSUE THREAD

**Issue by colltoaction** [lugfi/organizador-fiuba#28]
Para mantener la coherencia del código y hacer todo en un solo lenguaje (y no necesitar instalar PHP...) estaría bueno que el parser del PDF esté escrito en JavaScript. De ser difícil o irrelevante, cerrar este issue. 

Bloqueado por #27.

**Comment by edittler** [comment]
Hay herramientas en JS que hacen scraping de PDFs, comparto las que encontré:
* https://github.com/modesty/pdf2json
* https://github.com/nisaacson/pdf-extract

**Comment by fdnk** [comment]
En lugar de trabajar sobre un PDF, se puede utilizar una herramienta que convierta de PDF a TXT y trabajar sobre texto plano. Por ejemplo: pdftotext https://en.wikipedia.org/wiki/Pdftotext

**Comment by colltoaction** [comment]
Yo me inclinaría a usar una herramienta que haga como lo que propone @fdnk
pero en JavaScript. Prefiero hacer `npm install` y tener todo andando y no
andar buscando dependencias en el gestor de paquetes nativo de mi distro.
Seguro alguna de las que comenta Pampa tenga esa feature.

**Comment by edittler** [comment]
La primer opción que pasé aparentemente funciona sin dependencias externas ([pdf2json](https://github.com/modesty/pdf2json)) por lo que el scraper/parser sería puramente JS. Habría que probar. La segunda opción si dice requerir dependencias externas, se podría descartar.

**Comment by lucasp90** [comment]
Estoy de acuerdo en lo que dice @tinchou respecto a que sea en js, especialmente por si alguien quiere colaborar pero usa otro sistema operativo.

**Comment by AlexDaciuk** [comment]
Se esta haciendo la herramienta pdf -> custom csv en python
